src/patterns/data/default/entity_attribute.tsv

```diff
@@ -4100,3 +4100,291 @@ OBA:2050052	occipital lobe volume 	UBERON:0002021	occipital lobe	PATO:0000918	vo
 OBA:2050053	lentiform nucleus volume	UBERON:0002263	lentiform nucleus	PATO:0000918	volume
 OBA:2050097	serum amyloid A protein measurement	CHEBI:138152	serum amyloid A	PATO:0000070	amount
 OBA:2050098	serum paraoxonase/arylesterase 1 measurement	PR:000013021	serum paraoxonase/arylesterase 1	PATO:0000070	amount
+OBA:2060000	aggression towards inanimate objects	NBO:0000112	aggressive behavior towards inanimate objects	PATO:0000161	rate
+OBA:2060001	enhanced contextual conditioning behavior	NBO:0000223	contextual conditioning behavior	PATO:0000161	rate
+OBA:2060002	enhanced coordination	NBO:0000339	motor coordination	PATO:0000161	rate
+OBA:2060003	excessive tearing	GO:0070075	tear secretion	PATO:0000161	rate
+OBA:2060004	hyperactivity	NBO:0000338	kinesthetic behavior	PATO:0000161	rate
+OBA:2060005	impaired cued conditioning behavior	NBO:0000224	cued conditioning behavior	PATO:0000161	rate
+OBA:2060006	increased aggression	NBO:0000015	aggressive behavior	PATO:0000161	rate
```

Comment:
remove all values from the `defined_class_label` column. They are wrong, and DOS-DP will generate them automatically. Look here, it says "increased aggression". While the trait is about "rate of aggressive behaviour".

Reply:
(leave empty value)